cmd/apitodoc/main.go

```diff
@@ -35,7 +35,7 @@ func iterate(output io.Writer, data interface{}, indent int) {
 	if dataType == reflect.Slice || dataType == reflect.Map {
```

Comment:
There are some commented out statements in this function (lines 33, 44). Should these be removed?

Reply:
I didn't write them. But, it seems that line 33 is used in the commented out debug code in lines 36 and 41.
I don't see the point of line 44. I'll remove it.